tests/e2e/lib.sh

```diff
@@ -43,42 +43,48 @@ deploy_stackrox() {
 }
 
 # shellcheck disable=SC2120
-deploy_stackrox_with_custom_sensor() {
-    if [[ "$#" -ne 1 ]]; then
-        die "expected sensor chart version as parameter in deploy_stackrox_with_custom_sensor"
+deploy_stackrox_with_custom_central_and_sensor_versions() {
+    if [[ "$#" -ne 2 ]]; then
+        die "expected central chart version and sensor chart version as parameters in deploy_stackrox_with_custom_central_and_sensor_versions: deploy_stackrox_with_custom_central_and_sensor_versions <central chart version> <sensor chart version>"
     fi
-    target_version="$1"
-    setup_podsecuritypolicies_config
-
-    deploy_central
-
-    export_central_basic_auth_creds
-    wait_for_api
-    setup_client_TLS_certs "${2:-}"
-    record_build_info
+    ci_export CENTRAL_CHART_VERSION_OVERRIDE "$1"
+    ci_export SENSOR_CHART_VERSION_OVERRIDE="$2"
+    ci_export DEPLOY_STACKROX_VIA_OPERATOR "false"
+    ci_export OUTPUT_FORMAT "helm"
+    ci_export DISABLE_RHACS_IMAGE_REPOSITORY_PARAMS "true"
+    unset COLLECTOR_IMAGE_REPO
 
-    # generate init bundle
-    password_file="$ROOT/deploy/$ORCHESTRATOR_FLAVOR/central-deploy/password"
-    if [ ! -f "$password_file" ]; then
-        die "password file $password_file not found after deploying central"
-    fi
-    kubectl -n stackrox exec deploy/central -- roxctl --insecure-skip-tls-verify \
-        --password "$(cat "$password_file")" \
-      central init-bundles generate stackrox-init-bundle --output - 1> stackrox-init-bundle.yaml
+    helm repo add stackrox-oss https://raw.githubusercontent.com/stackrox/helm-charts/main/opensource
```

Comment:
Good point, I'll call it `temp-stackrox-oss-repo-should-not-see-me` as I see you did in `get_latest_helm_chart_versions.py` and do a `helm repo remove`, that should not clash with anyone.

Reply:
Looks like I have to rename it again to something short because `helm search repo` has a character limit in it's output and I need to regex to see if the correct charts are available. I'll name it "tmp-srox-compat" instead.